src/components/Table/head/__test__/selectableHeader.spec.js

```diff
@@ -3,54 +3,36 @@ import { mount } from 'enzyme';
+    it('should not render the PrimitiveCheckbox when maxRowSelection is 1', () => {
         const component = mount(<SelectableHeader maxRowSelection={1} />);
-        expect(component.find('InputCheckbox').exists()).toBe(false);
+        expect(component.find('PrimitiveCheckbox').exists()).toBe(false);
         expect(component.find('th[scope="col"]').exists()).toBe(true);
     });
     it('should set scope to "col" in th element', () => {
         const component = mount(<SelectableHeader />);
         expect(component.find('th').prop('scope')).toBe('col');
     });
-    it('should set name to the right value in InputCheckbox component', () => {
+    it('should set name to the right value in PrimitiveCheckbox component', () => {
         const component = mount(<SelectableHeader tableId="table-25" />);
-        expect(component.find('InputCheckbox').prop('name')).toBe('table-25-options');
+        expect(component.find('PrimitiveCheckbox').prop('name')).toBe('table-25-options');
     });
-    it('should set type to "checkbox" in InputCheckbox component', () => {
+    it('should set type to "checkbox" in PrimitiveCheckbox component', () => {
         const component = mount(<SelectableHeader />);
-        expect(component.find('InputCheckbox').prop('type')).toBe('checkbox');
+        expect(component.find('PrimitiveCheckbox').prop('type')).toBe('checkbox');
     });
-    it('should set checked to true in InputCheckbox component when bulkSelection is "all"', () => {
+    it('should set checked to true in PrimitiveCheckbox component when bulkSelection is "all"', () => {
         const component = mount(<SelectableHeader bulkSelection="all" />);
-        expect(component.find('InputCheckbox').prop('checked')).toBe(true);
+        expect(component.find('PrimitiveCheckbox').prop('checked')).toBe(true);
     });
-    it('should set checked to false in InputCheckbox component when bulkSelection is other than "all"', () => {
-        const values = ['none', 'some', '', 123, undefined];
-        values.forEach(value => {
-            const component = mount(<SelectableHeader bulkSelection={value} />);
-            expect(component.find('InputCheckbox').prop('checked')).toBe(false);
-        });
-    });
-    it('should set indeterminate to true in InputCheckbox component when bulkSelection is "some"', () => {
-        const component = mount(<SelectableHeader bulkSelection="some" />);
-        expect(component.find('InputCheckbox').prop('indeterminate')).toBe(true);
-    });
-    it('should set indeterminate to false in InputCheckbox component when bulkSelection is other than "some"', () => {
-        const values = ['none', 'all', '', 123, undefined];
-        values.forEach(value => {
-            const component = mount(<SelectableHeader bulkSelection={value} />);
-            expect(component.find('InputCheckbox').prop('indeterminate')).toBe(false);
-        });
-    });
```

Comment:
This three tests case needs to be converted:
- should set checked to false in InputCheckbox component when bulkSelection is "none"
- should set checked to "indeterminate" in InputCheckbox component when bulkSelection is "some"
- should set checked to true in InputCheckbox component when bulkSelection is "all"